tasks/e2e/cypress/integration/tutorial/tutorial.js

```diff
@@ -80,12 +80,7 @@ describe('The Redwood Tutorial - Golden path edition', () => {
     // https://redwoodjs.com/tutorial/getting-dynamic
     cy.writeFile(path.join(BASE_DIR, 'api/db/schema.prisma'), Step4_1_DbSchema)
     cy.exec(`rm ${BASE_DIR}/api/db/dev.db`, { failOnNonZeroExit: false })
-    cy.exec(`cd ${BASE_DIR}; yarn rw prisma migrate dev`, {
-      env: {
-        DATABASE_URL: 'file:./dev.db',
-        BINARY_TARGET: 'native',
-      },
-    })
+    cy.exec(`cd ${BASE_DIR}; yarn rw prisma migrate dev`)
```

Comment:
I believe we will now run into an interactive request to name the migration, which is not ideal. I'll check and update this line as necessary; most likely easy enough to pass `--name`

Reply:
Ok! I don't think I did when I tested it. I think prisma can automatically determine the name now and will only prompt you if it can't.

Reply:
Ah, forgot to update here. As is, it is _not_ interrupting the Cypress run. I have no idea why not because in manual use `migrate dev` prompts for migration every time. Not broken == not gonna fiddle